ExtensionHandler/Linux/src/AzureRM.py

```diff
@@ -790,6 +822,11 @@ def main():
       elif(input_operation == Constants.UPDATE):
         update()
 
+      try:
+        event_logger.log_new_event("extension_succeeded", "Extension successfully installed")
+      except Exception as e:
```

Comment:
There's no need to use `except Exception as e:` as you're not doing anything with `e`, thus `except Exception` is enough.

Reply:
Sure, changes made